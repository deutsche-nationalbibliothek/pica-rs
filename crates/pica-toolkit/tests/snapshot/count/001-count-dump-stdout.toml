bin.name = "pica-v1"
args = "count --skip-invalid dump.dat.gz"
status = "success"
stderr = ""
