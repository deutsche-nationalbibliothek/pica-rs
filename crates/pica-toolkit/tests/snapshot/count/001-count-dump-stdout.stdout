records: 6
fields: 295
subfields: 677
